### ISSUE TITLE
Support multiple configmaps for dashboards

### ISSUE THREAD

**Issue by metalmatze** [brancz/kubernetes-grafana#24]
After adding one one dashboard the configmap with the kubermatic-mixins is already too big.
We need a way to make sure that you can have mutliple configmaps.

**Comment by brancz** [comment]
I've been [dreading this day, but I knew it would come](https://github.com/brancz/kubernetes-grafana#roadmap). Unfortunately I don't think there is an easy functional way to do this in jsonnet (although my jsonnet foo is also not super strong). We should have a configured max bytes per configmap, then `manifest` each dashboard and "bin-pack" them returning a `ConfigMapList`. This is actually not unique for dashboard configmaps, so we should extract this into an extra library eventually, but let's first develop this in this repository and then extract it as step 2.

**Comment by metalmatze** [comment]
Sorry I brought it up. But I really need this now, as the kubernetes-mixins are almost filling a configmap... :smile: 
Do you have an idea on how to do the limiting stuff with jsonnet? I think having a way to specify which dashboards go into which configmaps also works. I additionally want to have multiple folders for my dashboards, one for kubernetes, one for prometheus stuff and then more for applications.
That's why I'd need to split them up manually anyway. 
For a start that would already be enough, at least for my use case.

**Comment by unguiculus** [comment]
FWIW, I have a custom Grafana Helm chart that simply creates separate Configmaps per dashboard without any bin-packing. It's simple and works. I don't see why I should try and squeeze them into as few Configmaps as possible.

**Comment by metalmatze** [comment]
@unguiculus, good point. Not sure why to do that either, now that I think about it. Purging might get a bit harder when one wants to remove all configmaps, but that's about it, right? :thinking:

**Comment by brancz** [comment]
Yeah I think I agree, thanks for bringing it up @unguiculus. Let's change it to do that (sad as I just finished a working version of a functional bin-packing in jsonnet :smile:, https://gist.github.com/brancz/d846b735c67de3df651dfe1f8dc01327).
